unit_tests/test_utils_issues.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+import pytest
+
+from sdcm.utils.issues import SkipPerIssues
+
+pytestmark = pytest.mark.integration
+
+
+def test_no_existing_issue(events):
+    with events.wait_for_n_events(events.get_events_logger(), count=1, timeout=10):
+        assert not SkipPerIssues(["ABC"], {'scylla_version': '2023.1.3'})
+
+    events_list = []
+    with events.get_raw_events_log().open() as events_file:
+        for line in events_file.readlines():
+            events_list.append(json.loads(line))
+
+    assert events_list[-1]['message'] == "couldn't parse issue: ABC"
+    assert events_list[-1]['severity'] == 'WARNING'
+
+
+def test_open_sources_open_issues():
+    assert SkipPerIssues(["http://github.com/scylladb/scylladb/issues/16206",
```

Comment:
It is fragile to depend on the status of the real issues of the alive projects.
Need either to use special project where we are not expecting the status changes or mock it.

Reply:
I've create special issues for it under qa-tasks, so it won't be based on real issues